back-end/www/tests/location_tests.py

```diff
@@ -0,0 +1,183 @@
+from basic_tests import BasicTest
```

Comment:
Thank you so much for writing the test cases! Before merging the code to the main branch, could you clean the code of the test cases, please? As an example, you can use "git diff" to check the changes that I made to the "answer_tests.py" file.